tests/cupy_tests/core_tests/test_raw.py

```diff
@@ -86,6 +97,15 @@ def setUp(self):
         self.mod2 = cupy.RawModule(_test_source2)
         self.mod3 = cupy.RawModule(_test_source3, ("-DPRECISION=2",))
 
+    def tearDown(self):
+        # To avoid cache interference, we remove cached files after every test
```

Comment:
A test should not clear the kernel cache: it affects too much.
How about temporarily changing the cache directory during the test?
In order for that, we will need a function to set the cache directory (can be private (prefix with `_`) for now).